altinn-client/src/main/java/no/difi/meldingsutveksling/AltinnWsClient.java

```diff
@@ -1,6 +1,7 @@
 package no.difi.meldingsutveksling;
 
 import com.google.common.collect.Lists;
+import com.sun.xml.ws.developer.JAXWSProperties;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1191)